Administrator Guide/Split brain and ways to deal with it.md

```diff
@@ -0,0 +1,68 @@
+# Split brain and the ways to deal with it
+
+### Split brain:
+Split brain is a situation where two or more replicated copies of a file become divergent. When a file is in split brain, there is an inconsistency in either data or metadata of the file amongst the bricks of a replica and do not have enough information to authoritatively pick a copy as being pristine and heal the bad copies, despite all bricks being up and online. For a directory, there is also an entry split brain where a file inside it can have different gfid/file-type across the bricks of a replica. Split brain can happen mainly because of 2 reasons:
+1. Due to network disconnect:
+Where a client temporarily loses connection to the bricks.
+2. Gluster brick processes going down or returning error:
+    - Server1 is down and server2 is up: Writes happen on server 2.
+    - Server1 comes up, server2 goes down (Heal not happened / data on server 2 is not replicated on server1): Writes happen on server1.
+    - Server2 comes up: Both server1 and server2 has data independent of each other.
+
+If we use the replica 2 volume, then there are high chances of ending up in split-brain.
+
+### Ways to deal with split brain:
+In glusterfs there are ways to resolve split brain. You can see the detailed description of automatically resolving split-brain [here](../Troubleshooting/heal-info-and-split-brain-resolution.md) and manual resolution [here](../Troubleshooting/split-brain.md). Moreover, there are ways which can reduce the chances of ending up in split-brain situations. They are:
+1. Replica 3 volume
+2. Arbiter volume
+
+Both of these uses the client-quorum option of glusterfs to avoid the split-brain situations.
+
+### Client quorum:
+This is a feature implemented in AFR to prevent split-brains in the I/O path for replicate/distributed-replicate volumes. By default, if the client-quorum is not met for a particular replica subvol, it becomes read-only. The other subvols (in a dist-rep volume) will still have R/W access. [Here](arbiter-volumes-and-quorum.md#client-quorum) you can see more details about client-quorum.

```

Comment:
I am not sure new users would know what AFR is. May be we should call it Automatic file replication module and in brackets you can call it "AFR here on"